### ISSUE TITLE
Possible Error in ARC-CO calculation

### ISSUE THREAD

**Issue by KelleyDrake** [ddrake/Integrated_Farm_Budget_Tool#212]
I was looking at my Champaign IL farm.   I have title acres of 475 for corn, 475 for beans and 50 for wheat.  Guarantee County Revenue from the FSA spreadsheet for non-irrigated is $1,090.47 for corn, $790.52 for soybeans and $481.96 for wheat.   Applying the 86% factor to the Guarantee County Revenue produces $937.80 for corn, $679.85 for soybeans and $414.49 for wheat.   

I run a sensitivity table and looking at the worst-case scenario in which revenues are WAY below the 86% of guaranteed revenue.  For example, at corn yield of 115 and price of $2.45 corn revenue is $281.8, far below the $937.80 guarantee.   That would generate a large payment of $656 if the payments were not subject to a CAP.     

The CAP is defined in the FSA spreadsheet as and equal to 10% of the Guarantee of $1,090.47 for corn (and NOT 10% of the 937.80 or (86% of $1,090.47).     Therefore, at very low prices and yields each of corn wheat and beans should be paying out at a rate equal to the maximum CAP and net of the 6.2% sequester.

Corn Cap of $109.05 x 475 = $51.80
Bean Cap of $79.05 x 475 = $37.55
Wheat cap of $48,29 x 50 = $2.41

The sum of those payments equals $91.76K.  After sequester of 6.2% that would equal $86.07K. 

This is what I would have expected IFBT to show in the sensitivity analysis at prices and yields that would max out the payments.   The IFBT sensitivity analysis, however, shows the whole farm max payment equal to $73K (considerably below the $86K that I would have expected).   

I think what might be happening is that the CAP is not being calculated correctly.   And that perhaps the 10% cap is being calculated by taking (10% x 86% of the Guaranteed Revenue) instead of (10% being of 100% of the Guaranteed revenue)?

For example, if I take the $91.76 total payments before sequester from above and I multiply them by 86% I get $78.91k.  After applying the 6.2% sequester that gets me to the $74.0k (very close to the $73k that is showing up in IFBT sensitivity table).

 Bottom line, for my Champaign County farm with all of the base acres in ARC-CO, I would have expected at very low prices and yields, corn, beans and wheat would max out payments.   Maxing those payments out at $91.76, ($86.07 after the 6.2% sequester).   So I would have expected to see $86k instead of $73k in the sensitivity table at very low prices and yields.    

Can you take a look at that?   I might be missing something obvious also.

**Comment by ddrake** [comment]
Are you considering the revenue shortfall?  The payment rate is the min(10% * bmk rev, rev shortfall).

**Comment by KelleyDrake** [comment]
Yes.  In those very low price yield scenarios the revenue shortfall is so large that the Max calculation should kick in.    I suspect I am missing something.   But I can't rconcile ifbt calculation and what I think should be correct payment for that champaign farm.



Sent from my Verizon, Samsung Galaxy smartphone



-------- Original message --------
From: Dow Drake ***@***.***>
Date: 1/5/24 4:50 PM (GMT-06:00)
To: ddrake/Integrated_Farm_Budget_Tool ***@***.***>
Cc: ***@***.***, Author ***@***.***>
Subject: Re: [ddrake/Integrated_Farm_Budget_Tool] Possible Error in ARC-CO calculation (Issue #212)


Are you considering the revenue shortfall? The payment rate is the min(10% * bmk rev, rev shortfall).

**Comment by ddrake** [comment]
I think you're assuming that there will be a wheat payment, but the sensitivity table is only looking at corn and beans.  I'm seeing a pre-sequestration amount of 0 for wheat.

**Comment by KelleyDrake** [comment]
That's weird.   At those very low prices and very low yields there should be a max wheat payment too. Along with max corn and max beans.



Sent from my Verizon, Samsung Galaxy smartphone



-------- Original message --------
From: Dow Drake ***@***.***>
Date: 1/5/24 5:08 PM (GMT-06:00)
To: ddrake/Integrated_Farm_Budget_Tool ***@***.***>
Cc: ***@***.***, Author ***@***.***>
Subject: Re: [ddrake/Integrated_Farm_Budget_Tool] Possible Error in ARC-CO calculation (Issue #212)


I think you're assuming that there will be a wheat payment, but the sensitivity table is only looking at corn and beans. I'm seeing a pre-sequestration amount of 0 for wheat.

**Comment by ddrake** [comment]
I think you're also assuming that net payment acres equal plc acres, but there's an 85% factor there.  I'm seeing the rates for corn and beans that you mentioned (i.e. 109.05 and 79.05), but I think you're using plc acres.

**Comment by ddrake** [comment]
I was wrong about there not being a wheat payment.  I'm getting a payment rate of 48.196 which is pretty close to the 48.29 that you said.  I think the only issue is that you weren't considering net payment acres.

**Comment by KelleyDrake** [comment]
That’s it.   You got it!   I forgot about the 15% reduction in acres to get to net payment acres.

**Comment by ddrake** [comment]
Ok closing this.

**Comment by KelleyDrake** [comment]
Thanks for chasing that down.  Sorry…I suspected the math was right and I was missing something.   Thanks.

**Comment by ddrake** [comment]
No worries – it's always good to check if something seems off, but it's just a bit more logic that most people can keep in their head.
